npm/pkg/dataplane/dataplane.go

```diff
@@ -107,6 +107,11 @@ func (dp *DataPlane) RunPeriodicTasks() {
 				// in Windows, does nothing
 				// in Linux, locks policy manager but can be interrupted
 				dp.policyMgr.Reconcile()
+
+				// the pod and namespace controllers won't retry failed calls to ApplyDataplane
+				if err := dp.ApplyDataPlane(); err != nil {
```

Comment:
Is there a possibility for us to applyDP when a diff controller is in between its CRUD event ?

Reply:
yes, but I'm not worried about this because we already have 3 controller threads that could call ApplyDataplane at any time

Reply:
now, we will fix this current behavior. We're now going to prevent another thread from applying changes in between a Controller's CRUD event